starlite/routes.py

```diff
@@ -89,7 +89,14 @@ def create_handler_kwargs_model(self, route_handler: BaseRouteHandler) -> Kwargs
         """
         dependencies = route_handler.resolve_dependencies()
         signature_model = get_signature_model(route_handler)
-        path_parameters = {p["name"] for p in self.path_parameters}
+        path_parameters = set()
+        for param in self.path_parameters:
+            param_name = param["name"]
+            if param_name in path_parameters:
+                raise ImproperlyConfiguredException(
+                    f"Duplicate path parameters should not be used. Duplicate parameter {{{param_name}}} found in path {self.path}"
```

Comment:
Is the `{{{...}}}` so that it is formatted like a path param?

How about:
`f"Duplicate parameter '{param_name}' detected in '{self.path}'."`